dts/bindings/pinctrl/nxp,s32ze-pinctrl.yaml

```diff
@@ -60,9 +60,7 @@ description: |
 
   Additionally, Safe Mode is always disabled (reset value) and configuration that
   only applies to LVDS pads, which are not supported, default to reset values:
```

Comment:
if the intention is to support LVDS pads, then `configuration that only applies to LVDS pads, which are not supported` must be removed

Reply:
updated it

Reply:
the reset values needs to still be mentioned. Only the part that LVDS pads are not supported must be removed/rephrased.

Reply:
I updated it